### ISSUE TITLE
Returns bool(false) when trying to get received messages

### ISSUE THREAD

**Issue by amankapoor** [Tustin/sarahah-php#1]
This is not working. Returns bool(false) when trying to get received messages.

**Comment by Tustin** [comment]
I'll have a look at it and see if I maybe messed something up. Their API responses are very vague so it could be hard to figure out what might be failing.

**Comment by Tustin** [comment]
I just tried it myself and it seems to be fine for me. Are you sure that your login credentials are correct?
http://i.imgur.com/XrCC7lp.png

**Comment by amankapoor** [comment]
Yes, you are able to do it. See my workspace. http://i.imgur.com/awCmkTt.jpg

May be I am doing something wrong. I followed your instructions though. Please help me find what's missing.

**Comment by amankapoor** [comment]
Yes, my login credentials are correct. I even changed my password to include only lowercase english alphabets and then tried it, but no success. Tried the whole process again in a new directory, but I still got bool(false).

I have sent you my php version and workspace screenshot above. Please have a look and let me know what can be done. Shall I try with your version of php? If so, share with me.

**Comment by Tustin** [comment]
Well I'm using PHP 5.6.25 so it shouldn't be caused by anything PHP related. Try doing a var_dump on the curl response inside both the get and post methods in agent.php. It might be due to you not having curl installed or something. It's hard to say since I can't reproduce it.
